PC02_Graffiti_Kanu.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Isha Kanu
+September 09, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+#=======Add your code here======
+#Red Hexagon shape
+up()
+goto(250,100)
+down()
+color("red")
+begin_fill()
+forward(50)
+right(45)
+forward(50)
+right(45)
+forward(50)
+right(45)
+forward(50)
+right(45)
+forward(50)
+right(45)
+forward(50)
+right(45)
+forward(50)
+right(45)
+forward(50)
+end_fill()
+
+# Black Warning Sign
+up()
+goto(265,85)
+down()
+pensize(3)
+color("yellow","black")
+begin_fill()
+right(135)
```

Comment:
Removing lines 59-66 to create a for loop
```suggestion

for i in range(2): #Repeats the line movements to create a rectangle shape.
    forward(20)
    right(90)
    forward(75)
    right(90)

```